### DISCUSSION TITLE
Custom login and SSO

### DISCUSSION THREAD

**Discussion by thirsty-bankroll** [Budibase/budibase#6590] [Ideas]
Hello guys,
It's an another message to ask for this feature.  I have to use budibase for internal and external app but right now I can't! I'm working for a company and it's a must have for us! I think it would be great way for bubibase to earn money.

My another idea would be to add AWS Cognito as a SSO provider!

**Comment by ryzam** [comment]
You can use Open ID plugin and i think it should working with any SSO provider support OpenID/OAuth2
